tests_v2/test_ssh_remote_shell_integration.py

```
@@ -0,0 +1,337 @@
+            _must(
+                term_program == "ghostty",
+                f"ssh-env should propagate TERM_PROGRAM=ghostty when AcceptEnv allows it, got: {term_program!r}",
+            )
+
+            term_program_version = _read_probe_payload(client, surface_id, "printf '%s' \"${TERM_PROGRAM_VERSION:-}\"")
+            _must(bool(term_program_version), "ssh-env should propagate non-empty TERM_PROGRAM_VERSION")
+
+            try:
+                client.close_workspace(workspace_id)
+            except Exception:
+                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)